Library/Homebrew/cli/named_args.rb

```diff
@@ -47,6 +259,8 @@ def to_formulae
           .returns(T::Array[T.any(Formula, Keg, Cask::Cask)])
       end
       def to_formulae_and_casks(only: parent&.only_formula_or_cask, ignore_unavailable: nil, method: nil)
```

Comment:
Can the logic in these be replaced by `convert`?

Reply:
Probably yes